packages/framework/src/Form/Admin/Domain/DomainFormType.php

```diff
@@ -26,14 +26,14 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'required' => false,
                 'file_constraints' => [
                     new Constraints\Image([
-                        'mimeTypes' => ['image/png', 'image/jpg', 'image/jpeg', 'image/gif'],
-                        'mimeTypesMessage' => 'Image can be only in JPG, GIF or PNG format',
+                        'mimeTypes' => ['image/png'],
+                        'mimeTypesMessage' => 'Image can be only in PNG format',
```

Comment:
Are we ok with the behavior change?

Reply:
I discussed this with @pk16011990 and yes, we are ok with this change :slightly_smiling_face: